### DISCUSSION TITLE
Update on the development branch

### DISCUSSION THREAD

**Discussion by kaiyux** [NVIDIA/TensorRT-LLM#1170] [Announcements]
Hi,

The TensorRT-LLM team is pleased to announce that we are pushing an update to the [development](https://github.com/NVIDIA/TensorRT-LLM/tree/main) branch (and the [Triton backend](https://github.com/triton-inference-server/tensorrtllm_backend/tree/main)) this February 27, 2024.

This [update](https://github.com/NVIDIA/TensorRT-LLM/pull/1168) includes:

* Model Support
  - Adding distil-whisper model support, thanks to the contribution from @Bhuvanesh09 in PR #1061
  - Migrate BLIP-2 examples to `examples/multimodal`
* Features
  - BART LoRA support (limited to the Python runtime)
  - Support `early_stopping=False` in beam search for C++ Runtime
  - Add logits post processor to the batch manager (see docs [here](https://github.com/NVIDIA/TensorRT-LLM/blob/main/docs/source/batch_manager.md#logits-post-processor-optional))
* Bug fixes
  - Multi-GPU fixes for multimodal examples #1003
  - Fix wrong `end_id` issue for Qwen #987
  - Fix a non-stopping generation issue #1118 #1123
* Documentation
  - Update precision support matrix for Whisper

Thanks,
The TensorRT-LLM Engineering Team

**Comment by akhoroshev** [comment]
> Add logits post processor to the batch manager (see docs [here](https://github.com/NVIDIA/TensorRT-LLM/blob/main/docs/source/batch_manager.md#logits-post-processor-optional))

It's very good and helps to customize for product tasks, thanks!

Please add the ability to configure the sampling pipeline (adding custom BaseSamplingLayer)
